scripts/env_wrapper.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -euo pipefail
 
-LIMA_VM="tee-builder"
+LIMA_VM="${LIMA_VM:-tee-builder}"
 
 # Check if Lima should be used
 should_use_lima() {
```

Comment:
Another case in which we should use lima could be "if lima is installed but nix isn't". Not asking you to implement this, but just wondering whether you think it's a good idea or whether it makes things too complicated.